mailpoet/assets/js/src/newsletters/send/congratulate/success_pitch_mss.tsx

```diff
@@ -3,10 +3,19 @@ import { MailPoet } from 'mailpoet';
 
 import { Heading } from 'common/typography/heading/heading';
 import { WelcomeWizardStepLayoutBody } from '../../../wizard/layout/step_layout_body.jsx';
-import {
-  Controls,
-  FreeBenefitsList,
-} from '../../../wizard/steps/pitch_mss_step';
+import { List } from '../../../common/typography/list/list';
+import { Button } from '../../../common';
```

Comment:
Suggestion: You can import directly from `common` like `import {something} from 'common'`, and defining an index file in `typography` folder should allow unifying importing `Heading` and `List`.

Reply:
Thanks for the suggestion. I implemented it in 53c78ce6ae5ff4fe1a1ffb62b7ed13d97e01c0e9.